src/vm-snapshot-form/finalFormTextArea.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { FormGroup, HelpBlock, Col, FormControl } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { inputObjectProps, metaObjectProps } from './finalFormPropTypes';
+
+const FinalFormTextArea = ({
+  meta,
+  input,
+  label,
+}) => (
+  <FormGroup validationState={meta.error ? 'error' : null}>
```

Comment:
DRY? maybe something like https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/forms/CommonForm.js ?

Reply:
This is actually something that i am wondering about myself. Current behavior is consistent with forms in manageiq-ui-classic, but i would also prefer something similar. I think we should talk about this more, and decide if we wan't different behavior in new forms.